src/overlays/actors/ovl_En_Knight/z_en_knight.c

```diff
@@ -6,6 +6,10 @@
 
 #include "z_en_knight.h"
 #include "z64shrink_window.h"
+#include "overlays/actors/ovl_Mir_Ray3/z_mir_ray3.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
+#include "objects/object_knight/object_knight.h"
+#include "prevent_bss_reordering.h"
```

Comment:
Can this go above all the other headers? Generally we like it first if possible.